### DISCUSSION TITLE
Is this project even alive?

### DISCUSSION THREAD

**Discussion by ghost** [LawnchairLauncher/lawnchair#3425] [General]
Hi, just a general query. The title should explain it all.

Is it safe to use the Lawnchair launcher in April 2023? I don't mind the bugs, I haven't faced any yet. But I am worried about the **security vulnerabilities** in this app.
The last update was in August of the previous year.

**Comment by RameshGokamalla** [comment]
I always face some problem with any version LAWNCHAIR launcher. Lag, Freeze, doesn't work sometimes, Had to clear the data and setup launcher again 😞. But, LAWNCHAIR 2 is feature packed launcher I 💕 that but, It frequently doesn't work. Debug versions of this launcher has less features than v2, I'm facing some problem with widgets resizing, widget fit.
I feel the same way as the title. But, team is working on it. It's been 3 years since play store version is updated. 🐢.
Main developer of this project did something wrong and left the project since then no proper updates. It's "till kottem" or something.

**Reply by ghost** [reply]
That's really sad to know 😞

**Comment by jmcglock** [comment]
According to their page:

"This repository is currently in maintenance mode. Most of our main developers are busy in real life, so only minor bug fixes/features and dependency updates will be made in this repo. Issues won't be prioritized as well.

If you want to volunteer your time to restore more active development of Lawnchair, please message us. We'll gladly accept your help.

Note that this is just temporary. Once the developers have free time, more active development will be resumed."

**Reply by TLCPlusPlus** [reply]
I hope it really is temporary. I just found out about this awesome app and it would suck for it to already be inactive from my point of view. I am using this as a testing ground to see if I would love the pixel launcher, and see if the pixel 8 will be a perfect fit for my phone use case.

**Reply by KTibow** [reply]
It is somewhat active, some maintainers are doing some coding.

**Comment by robw-nl** [comment]
Lawnchair no longer works flawlessly on my Pixel 7 Pro after the latest September 2023 Google Play and security update. It crashes during startup and the function to send the error log no longer works.  What a shame! I've removed it from my device :{

**Comment by tiagozip** [comment]
It's almost 2024... still nothing

**Reply by robw-nl** [reply]
I moved to Nova 8 premium beta. Works flawlessly and my battery life is much much longer now.

**Reply by KTibow** [reply]
@tiagorangel2011
> This repository is currently in maintenance mode. Most of our main developers are busy in real life, so only minor bug fixes/features and dependency updates will be made in this repo. Issues won't be prioritized as well.
If you want to volunteer your time to restore more active development of Lawnchair, please message us. We'll gladly accept your help.
Note that this is just temporary. Once the developers have free time, more active development will be resumed.

@wijhenke nobody asked

**Reply by robw-nl** [reply]
I volunteered, isn't that a great thing?

**Reply by KTibow** [reply]
You're advertising a product

**Reply by tiagozip** [reply]
@KTibow bruh, it's a useful suggestion

**Reply by KTibow** [reply]
How do I know they aren't a agent for Nova Launcher trying to get more $$$
/mostlyjoking

**Reply by robw-nl** [reply]
I love(d) Lawnchair btw, used it for quite a few years. Hope it gets picked up again!

**Comment by tiagozip** [comment]
Yeah, me too

A domingo, 8/10/2023, 6:43 da tarde, robwijhenke ***@***.***>
escreveu:

**Comment by SeanDylan** [comment]
Project is no longer maintain its mode that's what i read
